torch/fx/experimental/migrate_gradual_types/constraint_generator.py

```diff
@@ -861,7 +861,7 @@ def broadcasting_inference_rule(n: Node, symbols, constraints, counter):
                       BinConstraintD(0, my_output, op_leq)])
             return [c], counter
 
-    elif isinstance(n.args[1], Node) and (isinstance(n.args[0], int) or isinstance(n.args[1], float)):
+    elif isinstance(n.args[1], Node) and isinstance(n.args[0], (int, float)):
```

Comment:
This is a functional change, but I am pretty sure the old behavior is buggy since it's extremely unlikely for an object to inherit from both `Node` and `float`